models/staging/stg_iterable.yml

```diff
@@ -321,6 +323,22 @@ models:
       - name: _fivetran_synced
         description: "{{ doc('_fivetran_synced') }}"
 
+  - name: stg_iterable__list_user
+    description: Table containing the relationship between users and lists they belong to.
+    columns:
+      - name: source_relation
+        description: "{{ doc('source_relation') }}"
+      - name: _fivetran_user_id
+        description: Unique identifier of the list-user relationship record
```

Comment:
Should we note that this is a foreign key reference to `stg_iterable__user_history._fivetran_user_id`? The description -- though accurate -- doesn't make that relationship clear

Reply:
Updated this too!